packages/pwa-buildpack/lib/__tests__/cli-create-custom-origin.spec.js

```diff
@@ -1,6 +1,5 @@
 jest.mock('../Utilities/configureHost');
 jest.mock('../Utilities/loadEnvironment');
-jest.mock('word-wrap', () => x => x);
```

Comment:
This is mocked universally now, because it can produce different line breaks in different test environments, messing up snapshots.